modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManager.vue

```diff
@@ -160,29 +171,6 @@ export default {
       const verb = this.relationshipField ? 'Choose' : 'Manage';
       return `${verb} ${this.moduleLabels.plural}`;
     },
-    items() {
-      const items = [];
-      if (!this.pieces || !this.headers.length) {
-        return [];
-      }
-
-      this.pieces.forEach(piece => {
-        const data = {};
-
-        // Extra data for internal use
-        data.lastPublishedAt = piece.lastPublishedAt;
-
-        this.headers.forEach(column => {
-          data[column.name] = piece[column.name];
-        });
-
-        data._id = piece._id;
-
-        items.push(data);
-      });
-
-      return items;
-    },
```

Comment:
This didn't seem like it providing much. It was getting in the way of checking non-header doc properties, we never modified it, i just subbed in the `pieces` prop where it was used

Reply:
🆒 I think this is from when the table markup was in this component and the property was called `rows`.